pmda/rmsf.py

```diff
@@ -186,13 +187,13 @@ def _reduce(res, frame_result):
         # res[0]  sum of squares
         # res[1]  mean
         """
-        k = frame_result[0]
-        position = frame_result[1].positions
+        k = result_single_frame[0]
```

Comment:
Make your code more explicit with
```python
k, atoms, sumsq, means = result_single_frame
```
and then do things like
```python
positions = atoms.positions.astype(np.float64)
...
if len(res) == 0:
   res = [sumsq, means]
else:
   res[0] += (k / (k + 1)) * (position - res[1]) ** 2
   res[1] = (k * res[1] + position) / (k + 1)
```

This immediately tells someone what `res` looks like! Make your code readable by using meaningful variable names.